### PR TITLE
Add output to trello option

### PR DESCRIPTION
**Pull request by ringvold**
Adding support for Trello as an output option for the resulting teams.

For now it requires an api key and token and a board name.
Token and key can be supplied through environment variables (and `.env` file) and CLI options.

The api for the CLI can be discussed. This is a proposed first version.

### CODE REVIEW

File: cli/src/Main.elm [hunk 1]
```diff
@@ -4,52 +4,123 @@ import Array
...
-    = Generate String
+    = Sort SortOptions
+
+
+type alias SortOptions =
+    { fileName : String
+    , trello : Bool
+    , boardName : Maybe String
+    , trelloKey : Maybe String
+    , trelloToken : Maybe String
```
Comment (jorgengranseth) [inline cli/src/Main.elm:29]: Maybe TrelloIntegration? 🤔

File: cli/src/Main.elm [hunk 2]
```diff
@@ -225,12 +395,152 @@ playerParser index content =
...
+
+
+maybeOr : Maybe a -> Maybe a -> Maybe a
+maybeOr ma mb =
+    case ma of
+        Nothing ->
+            mb
+
+        Just _ ->
+            ma
```
Comment (jorgengranseth) [inline cli/src/Main.elm:534]: Maybe.Extra.orElse?

File: cli/src/Main.elm (hunk 1 above)
Comment (ringvold) [reply cli/src/Main.elm:29]: Record med trello-feltene?

File: cli/src/Main.elm [hunk 3]
```diff
@@ -99,28 +111,26 @@ init flags cliOptions =
...
+                        inlineOrEnvConfig =
+                            Maybe.map2 TrelloConfig trello.key trello.token
+                                |> ME.orElse initModel.trelloConfig
+                    in
+                    case inlineOrEnvConfig of
+                        Just config ->
+                            ( { initModel | outputIntegration = Trello config trello.boardName }
+                            , Cmd.batch
+                                [ readFile options.fileName
+                                ]
```
Comment (jorgengranseth) [inline cli/src/Main.elm:126]: Trenger du `batch` her?

File: cli/src/Main.elm [hunk 4]
```diff
@@ -99,28 +111,26 @@ init flags cliOptions =
...
+                            )
 
-                    _ ->
-                        ( initModel, printAndExitFailure "Can not find trello config. Exiting." )
+                        _ ->
```
Comment (jorgengranseth) [inline cli/src/Main.elm:129]: `Nothing` eksplisitt?

**Pull request merged**